run.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-# install packages 
-apt-get install -qqy pigpio
+apt-get install -qqy libpigpio1=1.79-1+rpt1 libpigpio-dev=1.79-1+rpt1 
+apt-get install pigpio
```

Comment:
Why is this on a separate line? Also, this package is the "transitional" package, so all it does is include ALL the other pigpio packages that have been split out. If this was causing conflicts, then add this (and pin the version as well) to the line above, so it's all one operation.